data/weird_very_short_3_mice/config.txt

```diff
@@ -0,0 +1,17 @@
+[1 dark]

```

Comment:
timeline

Reply:
I think this is going to be a huge issue for people providing data, because they have gotten used to the idea of naming timeline files config.txt